src/index.js

```diff
@@ -43,11 +43,55 @@ function displayComponent(elmnts) {
     }
   }
 }
-
 function focusInput(index) {
   hideComponent(document.querySelector(`.details${index}`));
   displayComponent(document.querySelector(`.delete${index}`));
 }
+function displayLineThroughComponent(elmntsParam) {
+  const test = elmntsParam.classList;
+  if (!test.contains('line_through')) {
+    test.toggle('line_through');
+  }
+}
+function removeLineThroughComponent(elmnts) {
+  if (elmnts.isArray) {
+    elmnts.forEach((elmnt) => {
+      const test = elmnt.classList;
+      if (test.contains('line_through')) {
+        test.remove('line_through');
+      }
+    });
+  } else {
+    const test = elmnts.classList;
+    if (test.contains('line_through')) {
+      test.remove('line_through');
+    }
+  }
+}
+function completeTask(index) {
+  hideComponent(document.querySelector(`.uncomplete${index}`));
+  displayComponent(document.querySelector(`.complete${index}`));
+  hideComponent(document.querySelector(`.details${index}`));
+  displayComponent(document.querySelector(`.delete${index}`));
+  displayLineThroughComponent(document.querySelector(`.input${index}`));
+  tasklist.localData[index - 1].setCompleted(true);
+  tasklist.refreshIndex();
+}
+
+function uncompleteTask(index) {
+  hideComponent(document.querySelector(`.complete${index}`));
+  displayComponent(document.querySelector(`.uncomplete${index}`));
+  hideComponent(document.querySelector(`.delete${index}`));
+  displayComponent(document.querySelector(`.details${index}`));
+  removeLineThroughComponent(document.querySelector(`.input${index}`));
+  tasklist.localData[index - 1].setCompleted(false);
+  tasklist.refreshIndex();
+}
+
+function clearCompletedTasks() {
+  tasklist.clearCompletedTasks();
+  refreshTodoList();
+}
```

Comment:
- [Optional] Consider adding your functions in separate modules and import them to your index.js file. This is best ES6 practice.